tests/migrations/test_commands.py

```diff
@@ -278,6 +278,21 @@ def test_regression_22823_unmigrated_fk_to_migrated_model(self):
         """
         call_command("migrate", "migrated_unapplied_app", stdout=six.StringIO())
 
+    @override_settings(
```

Comment:
See Jenkins, but I think this test is problematic, for example:
```
./runtests.py migrations django.contrib.auth
```
will give errors about `django_content_type` not existing. I'd expect these tests to be part of the tests for auth/contenttypes, not in `migrations`.